### PR TITLE
Issue 1428

### LINKED ISSUE #1428: refactor ground-failure to use sub components
- If landslides or liquefactions is pending then show both in a pending state.

### PR DESCRIPTION
**Pull request by jdbrown-USGS**
fixes #1428

### CODE REVIEW

File: src/app/ground-failure/summary-view/summary-view.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  styleUrls: ['./summary-view.component.scss'],
+  templateUrl: './summary-view.component.html'
+})
+export class SummaryViewComponent implements OnInit {
+  @Input()
+  product: any;
+
+  constructor() {}
+
+  ngOnInit() {}
```
Comment (emartinez-usgs) [inline src/app/ground-failure/summary-view/summary-view.component.ts]: Remove

File: src/app/ground-failure/pending.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+export class PendingPipe implements PipeTransform {
+  /**
+   * Checks to see if landslides and liquifaction are both in a pending state.
+   * @param product
+   *    Ground failure product
+   *
+   * @return boolean
+   *
+   */
+  transform(product: any): any {
```
Comment (emartinez-usgs) [inline src/app/ground-failure/pending.pipe.ts]: Be more robust here.
```javascript
let pending = false;
try {
  pending = (
    product.properties['landslide-alert'].toLowerCase() !== 'pending' &&
    product.properties['liquefaction-alert'].toLowerCase() !== 'pending'
  );
} catch (e) {
  pending = true; // Or false, whichever makes more sense on error
}

return pending;
```

File: src/app/ground-failure/summary-item/summary-item.component.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  @Input()
+  populationAlertColor: String;
+  @Input()
+  populationAlertValue: String;
+  @Input()
+  type: String;
+
+  constructor() {}
+
+  ngOnInit() {}
```
Comment (emartinez-usgs) [inline src/app/ground-failure/summary-item/summary-item.component.ts]: Remove this if empty.

File: src/app/ground-failure/summary-view/summary-view.component.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.summary {
+  max-width: 1440px;
```
Comment (ehunter-usgs) [inline src/app/ground-failure/summary-view/summary-view.component.scss]: This feels very wide, any reason that you changed it from `900px`. I just don't like it when the labels on the hazard bars are larger than their headers.

**Pull request merged**